scripts/restore_db.sh

```diff
@@ -10,5 +10,5 @@
 for db in "users" "api" "main" "logs" 
   do
     echo "Loading table $db"
-    python /code/manage.py loaddata /code/backup/$db.json
+    /usr/local/bin/python /code/manage.py loaddata /code/backup/$db.json
```

Comment:
You know this must be a path thing too! I need to read up more on cron - the default python in the container is /usr/local/bin/python, but I think you hit the nail on the head that cron doesn't have the same environment.

Reply:
You're right, but in the cron jobs we originally used /usr/bin/python, which is python2:
```
root@04cedfcd9f4a:/code# /usr/bin/python --version                                                                                         
Python 2.7.16                                                                                                                              
root@04cedfcd9f4a:/code# python --version                                                                                                  
Python 3.5.7              
```
So even with the correct PATH it would fail, I think.